src/Marten/Services/JsonNetSerializer.cs

```diff
@@ -13,24 +16,22 @@ public class JsonNetSerializer: ISerializer
         private readonly ArrayPool<char> _charPool = ArrayPool<char>.Create();
         private readonly JsonArrayPool<char> _jsonArrayPool;
 
-
-
-        private readonly JsonSerializer _clean = new JsonSerializer
+        private readonly JsonSerializer _clean = new()
```

Comment:
I'm not sure we use the containment operator anymore, so this might no longer be needed. Not 100% sure about that though.

Reply:
I tried to use `ToJson` instead of `ToCleanJson` but there were set of failing tests around querying interface types collection.